internal/cmd/bind.go

```diff
@@ -113,6 +113,13 @@ func (c *bindCmd) Run(fl *pflag.FlagSet) {
 		}
 	}
 
+	flog.Info("code-server --link is deprecated. While the servers will remain online,")
+	flog.Info("we are not releasing new features or bugfixes. A future code-server")
+	flog.Info("release will include a v2 with new features. If you would")
+	flog.Info("like early access, reach out on https://cdr.co/join-community")
+	flog.Info("")
```

Comment:
I was not sure if this was the best way to do multi-line messages with `flog`. I also noticed there is no "warning" level for logs (see cdr/flog#1)